src/main/mainWindow.ts

```diff
@@ -452,7 +453,7 @@ export async function createWindows() {
 
     mainWin.webContents.on("did-finish-load", () => {
         splash.destroy();
-        mainWin!.show();
+        if (!startMinimized) mainWin!.show();
 
         if (Settings.store.maximized && !isDeckGameMode) {
```

Comment:
this will likely fix it
```suggestion
        if (!startMinimized && Settings.store.maximized && !isDeckGameMode) {
```

Reply:
or better yet (thanks lewi)
```js
mainWin!.maximize();
mainWin!.hide();
```

Reply:
@lewisakura  Please verify if it works now. My window manager might not have the maximize option, but during testing, when I manually set it to maximized state, it functioned correctly.

Reply:
The underlying issue is now resolved, however this will cause a flicker which is an unintended side effect. A better solution would be to not show the window as usual, and then when the window is first brought up check if it should be maximised or not. You can use the [BrowserWindow show event](https://www.electronjs.org/docs/latest/api/browser-window#event-show) for this.